data/tabular/nomad_structure/meta.yaml

```
@@ -0,0 +1,126 @@
+---
+name: nomad-structure
+description: |-
+    A subset from NOMAD dataset, which is a database of DFT computed results of materials.
+    This subset consists of cif structures of around 0.5 million bulk stable materials and their geometric and structural information.
+    All materials in this dataset are modeled using Density Functional Theory using GGA functional.
+targets:
+    - id: density
+      description: Density of the material
+      units: Kg/m^3
+      type: continuous
+      names:
+          - noun: density of the material
```

```suggestion
          - noun: density of the material
```

in your templates, you already use "of the material". If you have it here, too, it will appear twice in some samples